api/app/schemas/bookings/exam_schema.py

```diff
@@ -37,7 +37,6 @@ class Meta:
     exam_type_id = fields.Int()
```

Comment:
Looks like there is going to be an unused import for `InvigilatorSchema` in this file